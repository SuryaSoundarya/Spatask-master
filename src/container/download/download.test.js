import React from 'react';
import { shallow } from 'enzyme';
import Download from './index';

describe('Download', () => {
  it('should render correctly ', () => {
    const component = shallow(<Download />);
    expect(component).toMatchSnapshot();
  });
});