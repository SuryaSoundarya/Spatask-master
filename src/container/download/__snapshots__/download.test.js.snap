// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Download should render correctly  1`] = `ShallowWrapper {}`;
